src/migration/service/migrator.ts

```diff
@@ -3,8 +3,12 @@ import * as path from 'path';
 import { glob, exists } from '../../util/fs';
 import { resolveFile } from '../../service/sqlRunner';
 import SqlMigrationEntry from '../domain/SqlMigrationEntry';
+import FileExtensionTypes from '../../enum/FileExtensionTypes';
+import JavaScriptMigrationEntry from '../domain/JavaScriptMigrationEntry';
 
-const FILE_PATTERN = /(.+)\.(up|down)\.sql$/;
+const FILE_PATTERN_JS = /(.+).(js)$/;
```

Comment:
Isn't this the same? Why would we want to use capture group for the extension?

```suggestion
const FILE_PATTERN_JS = /(.+).js$/;
```